src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,173 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    static class Node<T> {
+
+        private T value;
```

Comment:
Don't use getters and setters in class Node. Since it's going to be an inner class, you can directly access its private fields within the enclosing class.

Reply:
there are no getters and setters in my class Node...